openpype/hosts/maya/plugins/publish/validate_look_color_space.py

```diff
@@ -0,0 +1,25 @@
+from maya import cmds
+
+import pyblish.api
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidateMayaColorSpace(pyblish.api.InstancePlugin):
+    """
+        Check if the OCIO Color Management and maketx options
+        enabled at the same time
+    """
+
+    order = ValidateContentsOrder
+    families = ['look']
+    hosts = ['maya']
+    label = 'Maya Color Space'
```

Comment:
I feel like this label is too generic - it doesn't actually validate any color space. The same goes for how it's listed in the OpenPype settings.

It should be clear that this relates at least to _Color Management_ and _maketx_
Would labeling it like `label = "Color Management with maketx" make more sense?